apps/portals/src/configurations/elportal/synapseConfigs/handleUpsetPlotClick.tsx

```diff
@@ -0,0 +1,21 @@
+import { Query } from '@sage-bionetworks/synapse-types'
+import { cohortBuilderFilesSql } from '../resources'
+
+export const handleUpsetPlotClick = (selection: any) => {
+  const { name } = selection
+  const query: Query = {
+    sql: cohortBuilderFilesSql,
+    selectedFacets: [
+      {
+        concreteType:
+          'org.sagebionetworks.repo.model.table.FacetColumnValuesRequest',
+        columnName: 'Assays',
+        facetValues: [name],
+      },
+    ],
+    // additionalFilters: [filter], //technically,
+  }
+  window.location.assign(
+    `/Explore/Data%20by%20Files?QueryWrapper0=${JSON.stringify(query)}`,
+  )
```

Comment:
Could consider using SRC's [generateEncodedPathAndQueryForSelectedFacetURL](https://github.com/Sage-Bionetworks/synapse-web-monorepo/blob/main/packages/synapse-react-client/src/components/QueryWrapper/generateEncodedPathAndQueryForSelectedFacetURL.ts), if you'd like! Something like -- 

```suggestion
  const url = generateEncodedPathAndQueryForSelectedFacetURL(
    '/Explore/Data%20by%20Files',
    cohortBuilderFilesSql,
    [{facet: 'Assays', facetValue: name}]
  window.location.assign(url)
```